docs/introduction/concepts.md

```diff
@@ -31,3 +31,27 @@ Producing and exchange olive oil and coffee, even more simplified.
 ### ... to thinking bigger.
 
 ![river flow](https://rawgit.com/valueflows/valueflows/master/release-doc-in-process/vf-river.png)
+
+### Traversing value flows
+
+Value flows can be traversed forwards ("tracking") or backwards ("tracing").  Often people use the term "provenance" when looking backwards towards the source of some resource, sometimes when a problem emerges (as in a disease outbreak from food), sometimes to know the quality of the resource (as in wanting local humanely produced food with lower ecological impact).
+
+The data structures of ValueFlows provide the ability to trace or track any value flow, no matter how long or complex:
+
+* For an EconomicResource:
+    * after: EconomicEvents that are inputs to Processes or Transfers
+    * before: EconomicEvents that are outputs from Processes or Transfers
+* For a Process or Transfer:
+    * after: EconomicEvents that are outputs
+    * before: EconomicEvents that are inputs
+* For an EconomicEvent:
+    * after: 
+        * a Process or Transfer to which it is an input, or
+        * an EconomicResource which it affected as the output of a Process or Transfer
+    * before:
+        * a Process or Transfer from which is is an output, or
+        * if it is an input to a Process or Transfer, the EconomicResource which it affects.
+
+When the same economic resource is both input and output of a process, and more than one resource is involved, the tracking identifier is used.  And example of this would be partial load shipments, where many separate shipments are included in the same shipment process.
```

Comment:
pretty much on of the scenarios discussed in #392 which also shows variant where we have different resource on input and output but they need to keep distinct *provenance anchor* / *processing transaction id* / *tracking id*, i think we should remove that *same economic resource is both input and output* and just keep something like *both input and output reference the same tracking (processing transaction) identifier*

Reply:
I very much disagree with having the provenance anchor (either a process or transfer) as sub to the main process.  And I didn't understand your suggested sentence.  I tried to clarify the text without implying more than what we have.  See if this is OK, or if I am going too much in "my" direction without taking into account your concerns!  Will continue to fix per discussion...

Changed to:
```
When there is more than one economic resource going from input to output of a process without transformation of that resource (except for location), the tracking identifier is used.
```